modules/helm/format.go

```diff
@@ -0,0 +1,81 @@
+package helm
+
+import (
+	"fmt"
+	"path/filepath"
+	"testing"
+
+	"github.com/gruntwork-io/gruntwork-cli/collections"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+	"github.com/stretchr/testify/require"
+)
+
+// FormatSetValuesAsArgs formats the given values as command line args for helm using set (e.g of the format
+// --set key=value)
+func FormatSetValuesAsArgs(setValues map[string]string) []string {
```

Comment:
Should this be public?

Reply:
Made private since I don't see anyone using this directly.